src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/process/DefaultProcessRunner.java

```diff
@@ -77,19 +78,15 @@ public void run(String[] command) throws ProcessRunnerException {
       }
       processBuilder.command(makeOsSpecific(command));
 
-      synchronized (this) {
```

Comment:
In the build plugins, there isn't really any re-use, right? We create a new instance of CloudSdk every time. This could be a little less straight-forward in the IDE plugins, but still doable.